builtin/providers/aws/resource_aws_appautoscaling_policy.go

```diff
@@ -47,16 +47,16 @@ func resourceAwsAppautoscalingPolicy() *schema.Resource {
 				Required: true,
 			},
 			"scalable_dimension": &schema.Schema{
-				Type:     schema.TypeString,
-				Optional: true,
-				Default:  "ecs:service:DesiredCount",
-				ForceNew: true,
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validateAppautoscalingScalableDimension,
 			},
 			"service_namespace": &schema.Schema{
-				Type:     schema.TypeString,
-				Optional: true,
-				Default:  "ecs",
-				ForceNew: true,
+				Type:         schema.TypeString,
```

Comment:
If we are changing these from Optional to required, then we may need to verify that the default value from before `ecs` is actually correct - otherwise we are breaking backwards compatibility